modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -1,66 +1,79 @@
-= Usage and Pricing Metrics
+// tag::intro[]
+Mule runtime reports track your use of Mule messages, Mule flows, and data throughput. 
 
-MuleSoft captures usage data for Mule flows, Mule messages, and data throughput, but not all of this data is aggregated in usage reports. 
+To track Mule message usage, the runtime report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given day or month by business group, environment, and application. 
 
-For select customers in the US control plane, MuleSoft offers a pricing and packaging model for Anypoint Platform that allots a number of Mule flows, Mule messages, and data throughput (measured as network I/O bytes). Not all types of Mule flows and messages count toward the allotments in a package. 
+To calculate usage, MuleSoft meters and aggregates the number of messages on a daily and monthly basis. After a message is triggered, the report does not track changes to the message because the message is processed within the application’s flows.
 
-== Metrics Collected in Anypoint Platform
+A Mule flow is a sequence of logical operations configured within the XML `<flow/>` element of a Mule application. The runtime report tracks the Mule flows within a deployed and running Mule application that contains a Mule event source or APIkit route requests. This report doesn't track other Mule flows. You can view the maximum number of Mule flows in a given day or month in the usage report. 
 
-Anypoint Platform captures usage metrics across products, but not all metrics are billable or reflected in usage reports.
+To calculate usage, MuleSoft tracks the number of Mule flows for all business groups, environments, and applications. The maximum number of Mule flows within a day or month is identified based on the peak hour across the day or month. For the detailed breakdown, MuleSoft shows the peak hour usage per business group, environment, and application. 
 
-[cols=3*]
-|===
-|Product |Metric Description |Usage Details
+Data throughput is all the network I/O bytes produced by the infrastructure that starts and runs the Mule runtime engine that runs a Mule application. You can view the total amount of GB usage in a given day or month by business group, environment, and application. 
```

Comment:
Any way to simplify? 
Data throughput is the network I/O bytes produced by the infrastructure that runs the Mule runtime engine running the Mule application.